Core/StatisticsLoader.swift

```diff
@@ -134,6 +134,7 @@ public class StatisticsLoader {
     public func storeUpdateVersionIfPresent(_ atb: Atb) {
         if let updateVersion = atb.updateVersion {
             statisticsStore.atb = updateVersion
+            statisticsStore.variant = nil
```

Comment:
do we know for sure that update variant doesn't contain atb?

Reply:
Yes, see https://dub.duckduckgo.com/duckduckgo/ddg/blob/bttf/lib/DDG/Util/ATB.pm#L127

If you want to parse out the update version and apply the atb and variant separately to remain future proof then that's fine, but my understanding is this won't change as there's no value to keeping the variant (the whole point is to make small buckets of clients a little bigger in order to remain anonymous).